_posts/2023-09-25-announcing-duckdb-090.md

```diff
@@ -0,0 +1,151 @@
+* Index Improvements
+* DuckDB-WASM Extensions
+* Extension Auto-Loading
+* Improved AWS Support
+* Iceberg Support
+* Azure Support
+* PySpark-Compatible API
+
+Below is a summary of those new features with examples, starting with a change in our SQL dialect that is designed to produce more intuitive results by default.
+
+
+#### Breaking SQL Changes
+
+[**Struct Auto-Casting**](https://github.com/duckdb/duckdb/pull/8942). Previously the names of struct entries were ignored when determining auto-casting rules. As a result, struct field names could be silently renamed. Starting with this release, this will not throw an exception instead.
+
+```sql
+CREATE TABLE structs(s STRUCT(i INT));
+INSERT INTO structs SELECT {'k': 42};
+-- Mismatch Type Error: Type STRUCT(k INTEGER) does not match with STRUCT(i INTEGER). Cannot cast STRUCTs with different names
+```
+
+#### Core System Improvements
+
+**[Out-Of-Core Hash Aggregates.](https://github.com/duckdb/duckdb/pull/7931)** When working with large data sets, memory management is always a potential pain point. By using a streaming execution engine and buffer manager, DuckDB supports many operations on larger than memory data sets. DuckDB also aims to support queries where *intermediate* results do not fit into memory by using disk-spilling techniques.
+
+In this release support for disk-spilling techniques is further extended through the support for out-of-core hash aggregates. Now hash tables constructed during `GROUP BY` queries or `DISTINCT` operations that do not fit in memory due to the large amount of groups will spill data to disk instead of throwing an out-of-memory exception. Due to clever use of radix partitioning, performance degradation is gradual and performance cliffs are avoided. Only the subset of the table that does not fit into memory will be spilled to disk.
+
+|  memory limit | runtime   |
+|:-------------:|:---------:|
+| 6.0GB         | 6.84s     |
+| 5.0GB         | 7.39s     |
+| 4.0GB         | 9.04s     |
+| 3.0GB         | 8.76s     |
+| 2.0GB         | 8.08s     |
+| 1.0GB         | 10.27s    |
+
+**Window Function Performance Improvements ([#7831](https://github.com/duckdb/duckdb/pull/7831), [#7996](https://github.com/duckdb/duckdb/pull/7996), [#8050](https://github.com/duckdb/duckdb/pull/8050), [#8491](https://github.com/duckdb/duckdb/pull/8491)).** This release features many improvements to the performance of Window functions due to improved vectorization of the code, more re-use of partial aggregates and improved parallelism through work stealing of tasks. As a result, performance of [Window functions has improved significantly, particularly in scenarios where there are no or few partitions](https://github.com/duckdb/duckdb/issues/7809#issuecomment-1679387022).
+
+```sql
+SELECT
+    SUM(driver_pay) over (
+        ORDER BY dropoff_datetime ASC
+        RANGE BETWEEN
+        INTERVAL 3 DAYS PRECEDING AND
+        INTERVAL 0 DAYS FOLLOWING
+    )
+FROM tripdata
```

Comment:
```suggestion
FROM tripdata;
```
Can tripdata be made available over https somewhere on the website (unsure how big the dataset is). So that the query becomes runnable.